acquia_lift.module

```diff
@@ -389,6 +389,12 @@ function acquia_lift_init() {
       acquia_lift_unset_nav_message();
```

Comment:
I'm wondering if this stuff needs to get deleted (not necessarily as part of this story... I'm just fairly sure it's not even used anymore)